lir/plotting/expected_calibration_error.py

```diff
@@ -48,17 +46,21 @@ def plot_ece(
         indicating both ends of the range on the x-axis)
     """
     llrs = llrdata.llrs
+    labels = llrdata.labels
+    if labels is None:
+        raise ValueError('LLRData must contain labels to plot ECE.')
 
     log_prior_odds = np.arange(*log_prior_odds_range, 0.01)
     prior_odds = np.power(10, log_prior_odds)
 
     # plot reference
-    ax.plot(
-        log_prior_odds,
-        calculate_ece(np.ones(len(labels)), labels, odds_to_probability(prior_odds)),
-        linestyle=':',
-        label='reference',
-    )
+    if ax.get_legend() is None or 'reference' not in [text.get_text() for text in ax.get_legend().get_texts()]:
+        ax.plot(
+            log_prior_odds,
+            calculate_ece(np.ones_like(labels), labels, odds_to_probability(prior_odds)),
+            linestyle=':',
+            label='reference',
+        )
```

Comment:
Je zou er ook een vlaggetje van kunnen maken, met default `True`, of veroorzaakt dat andere problemen?